### PR TITLE
Added flake8, fixed syntax, flattened package structure

### PR DESCRIPTION
**Pull request by amureki**
@etianen 
I've decided to split my work into two PR's, otherwise changes are not easily readable and understandable.

Feel free to comment or make suggestions.
If everything would be fine in #186 (when it will be merged), I'll rebase this one on master, or feel free to do it by yourself if you want. 🙃

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,37 +1,30 @@
...
-  - DJANGO=django==1.9.9 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
-  - DJANGO=django==1.10
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.postgresql" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
 matrix:
   fast_finish: true
+env:
+  matrix:
+    - WORKER=python
+    - WORKER=flake8
```
Comment (etianen) [inline .travis.yml]: I'd make things nice and simple. Just run flake 8 as another tox env. There's very little to be gained running things in parallel.

File: .travis.yml [hunk 2]
```diff
@@ -1,37 +1,30 @@
...
   - postgresql
   - mysql
 install:
-  - travis_retry pip install $DJANGO
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql" ]] ; then travis_retry pip install psycopg2; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.postgresql_psycopg2" ]] ; then travis_retry pip install psycopg2; fi
-  - if [[ "$DB_ENGINE" == "django.db.backends.mysql" ]] ; then travis_retry pip install mysqlclient ; fi
-  - travis_retry pip install -e .
+  - pip install tox
+  - pip install flake8
```
Comment (etianen) [inline .travis.yml]: Just get tox to install flake8.

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist =
+    {py27,py34,py35}-django{18,19,110}-{sqlite,postgres,mysql}
```
Comment (etianen) [inline tox.ini]: Add `flake8` as another env.

A good reference tox.ini that works well is:

https://github.com/etianen/django-reversion/blob/master/tox.ini

It's pretty much identical to how you've done things. :)

**Comment by amureki** [comment]
@etianen yeah, this one was based on previous PR, that's why it had tox commit also. :)

But everything is totally fine, thanks for merging!

**Comment by luzfcb** [comment]
@etianen one side effect of rename directory in git, is to be forced to use `git log` with `--follow` parameter, if you need see the change history of a file and not being able to do this via the github UI

```
git log watson/models.py
git log --follow watson/models.py
```

**Comment by etianen** [comment]
@luzfcb - Yeah, I've been putting off the directory rename for a while, since it has a few minor side-effects, and I was lazy. :)

On the whole, it's better this way, though!

**Pull request merged**